server/plugins/task-plugin-api/src/main/java/com/oceanbase/odc/plugin/task/api/partitionplan/datatype/TimeDataType.java

```diff
@@ -37,8 +41,13 @@ public class TimeDataType extends GeneralDataType {
     public static final int MINUTE = 0x10 | HOUR;
     public static final int SECOND = 0x20 | MINUTE;
```

Comment:
Why `MONTH = 0x2 | YEAR`? What is the purpose of such a design? Is it used for removing excess precision?

Reply:
partition plan is designed for time type, but the precision of a time type is different, eg.
1. date's precision is only up to year, month and day.
2. datetime's precision can be up to hour, minute and second.

so that this field is used to indicate the precision of the time type. There is a priori knowledge here: High precision must include low precision, so I adopted this design to express the inclusion relationship between precisions.

> Is it used for removing excess precision?

yes

Reply:
I got you!